### ISSUE TITLE
🐛 [Bug]

### ISSUE THREAD

**Issue by ulfgebhardt** [gradido/gradido#877]
## 🐛 Bugreport

![IMG_0580](https://user-images.githubusercontent.com/1238238/134228398-edbf35ca-cbe1-4141-9d39-5635e234cba4.jpeg)

Passwort reset does produce errors in unknown cases. This is from the Support.

Furthermore there is no special char requirement on the PW field (but works)

- [ ] Display proper Errors to the User, we could interpret from a screenshot
- [ ] Find the error and fix it
- [ ] Do more password checks and align them - we should have the same password requirements everywhere (why does the login server accept this?)

**Comment by Mogge** [comment]
@einhornimmond Am I right when I have a whitelist of allowed chars like this:
`A-Za-z0-9@$!%*?&+_-`

**Comment by einhornimmond** [comment]
Yes but I think about changing the allowed chars to open up for every possible utf-8 char.
